shell/components/form/GitPicker.vue

```diff
@@ -73,58 +73,14 @@ export default Vue.extend<Data, any, any, any>({
     this.onSearchBranch = this.searchForResult(this.searchBranch);
   },
 
-  mounted() {
-    if (!this.value) {
-      return;
-    }
-
-    if (this.value.sourceData) {
-      // API calls data - from cache
-
-      this.selectedAccOrOrg = this.value.usernameOrOrg;
-      this.selectedRepo = this.value.repo;
-      this.selectedBranch = this.value.branch;
-      this.selectedCommit = { sha: this.value.commit };
-
-      this.repos = this.value.sourceData.repos;
-      this.branches = this.value.sourceData.branches;
-      this.commits = this.value.sourceData.commits;
-    } else {
-      // API calls data - from remote
-      this.selectedAccOrOrg = this.value.usernameOrOrg;
-      if (this.selectedAccOrOrg) {
-        this.fetchRepos()
-          .then(() => {
-            if (this.repos.length && !this.hasError.repo) {
-              const { name, id } = this.repos.find((r: any) => r.name === this.value.repo?.name) || {};
-
-              this.selectedRepo = { name, id };
-
-              return this.fetchBranches();
-            }
-          })
-          .then(() => {
-            if (this.branches.length && !this.hasError.branch) {
-              this.selectedBranch = this.value.branch;
-
-              return this.fetchCommits();
-            }
-          })
-          .then(() => {
-            if (this.commits.length && this.commits.find((c: commit) => c.sha === this.value.commit || c.id === this.value.commit)) {
-              this.selectedCommit = { sha: this.value.commit };
```

Comment:
This line hasn't made it over to the new location. It means the top commit is always selected regardless of whether it's the correct one (see `richard/rc-github` app. `chatty-app` commit deployed, but `Create README.md Added README` selected)